dbschema/seeds/007.language-engagements.edgeql

```diff
@@ -0,0 +1,66 @@
+with
+  engagementsJson := to_json('[
+   {
+      "status": "InDevelopment",
+      "startDateOverride": "2020-04-01",
+      "endDateOverride": "2020-06-30",
+      "language": "English",
+      "lukePartnership": true,
+      "openToInvestorVisit": true,
```

Comment:
I probably wouldn't call out these two here in seeding. They aren't that significant.

Reply:
I guess we wouldn't have to, but I thought the point was to have a variety of data already available for us in our locals.  So these would be baked in as options?

Reply:
Definitely on variety. I guess my point was we never really care about these. The things we look at a lot, and would want to have variety, are mainly status, products, progress reports.

Reply:
I'll concede; if we ever find ourselves having to continually set these on our locals to do troubleshooting somebody can just add them back in.